integreat_cms/api/v3/pages.py

```diff
@@ -212,20 +228,26 @@ def children(request, region_slug, language_slug):
         # like in wordpress depth = 0 will return no results in this case
         depth = depth - 1
     result = []
+    # pylint: disable=unused-variable
+    public_region_pages, skipped_pages = request.region.pages.filter(
+        explicitly_archived=False, tree_id__in=[page.tree_id for page in root_pages]
+    ).cache_tree(archived=False, language_slug=language_slug)
```

Comment:
Could you add the `language_slug=language_slug` argument to the "all pages" endpoint as well? Currently, the endpoint throws a 404 if there exists at least one page with a non-archived parent page without public translation. In theory, we could also add this special edge case to the tests, but not urgent I think, let's add this later.

Reply:
Sure.